app/(datasets)/data-catalog/page.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { getDatasets } from 'app/blog/utils/mdx';
+import { Suspense } from 'react';
+import Catalog from './catalog';
+
+export default function Page() {
+  const posts: any[] = getDatasets();
+
+  const transformData = () => {
```

Comment:
I see `transformData` repeated here and there. Wouldn't it be better to consolidate this function into `getDatasets` function so it doesn't have to be repeated and spread out?

Reply:
great catch, created an `app/utilities/` dir